queries.sql

```diff
@@ -1,3 +1,15 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name, date_of_birth  FROM animals WHERE name='Agumon' OR name='Pikachu';
```

Comment:
- [ ] Please add the query to list the following requirement, this is a project requirement. After that please update the screenshots 👇🏽 

> List the name of all animals that are neutered and have less than 3 escape attempts.